f/export/postgres_to_geojson/postgres_to_geojson.py

```diff
@@ -0,0 +1,136 @@
+# requirements:
+# psycopg2-binary
+
+import json
+import logging
+import os
+
+import psycopg2
+
+# type names that refer to Windmill Resources
+postgresql = dict
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def conninfo(db: postgresql):
+    """Convert a `postgresql` Windmill Resources to psycopg-style connection string"""
+    # password is optional
+    password_part = f" password={db['password']}" if "password" in db else ""
+    conn = "dbname={dbname} user={user} host={host} port={port}".format(**db)
+    return conn + password_part
+
+
+def main(
+    db: postgresql,
+    db_table_name: str,
+    storage_path: str = "/frizzle-persistent-storage/datalake/export",
+):
+    data = fetch_data_from_postgres(conninfo(db), db_table_name)
+
+    feature_collection = format_data_as_geojson(data)
+
+    save_file(feature_collection, db_table_name, storage_path)
```

Comment:
One day the quantity of data will be too much to hold into memory, so we will want to adapt a streaming solution.  No need to touch it now tho.

ALSO - I wonder if this script could be simplified to a thin wrapper over an `ogr2ogr` call.  There are postgres and geojson drivers for ogr2ogr.  I don't know if having just dropped the PostGIS extension makes the postgres driver a non-starter, but we could always add it back.

Reply:
Interesting. From a cursory read of the documentation https://gdal.org/en/stable/drivers/vector/pg.html it does seem that some ogr2ogr operations are supported without PostGIS installed. 

I propose we keep this in mind for a future reconsideration of how we handle geospatial db data across the stack, writ large.

Reply:
>One question: Once you remove them from frizzle.meta.yaml, does that actually end up changing the permissions in a way we don't want? I'd imagine it could.

So these lines get added when you run `wmill sync pull`. When I then remove the lines, and run `wmill sync push`, I see no impact on permissions on the server: I'm able to run, schedule, and edit scripts all the same, schedules are still up, and I see no changes in the UI otherwise. Maybe this is different for a regular admin (not superadmin). But for now I don't see a reason to keep these hard-coded.